Kudu.Core/Environment.cs

```diff
@@ -95,7 +97,20 @@ public class Environment : IEnvironment
             _siteExtensionSettingsPath = Path.Combine(SiteRootPath, Constants.SiteExtensionsCachePath);
             _diagnosticsPath = Path.Combine(SiteRootPath, Constants.DiagnosticsPath);
             _locksPath = Path.Combine(SiteRootPath, Constants.LocksPath);
-            _sshKeyPath = Path.Combine(rootPath, Constants.SSHKeyPath);
+
+            string siteName = System.Environment.GetEnvironmentVariable("APP_POOL_ID");
+            siteName = NormalizeSiteName(siteName);
```

Comment:
Can those two lines be moved inside the `else` block, since `siteName` is not used at all on Windows?